### PR TITLE
Add a script to easily delete a community/group from a homeserver

### PR DESCRIPTION
**Pull request by Matombo**
I copied the logic of nuke-room-from-db.sh and altered it to delete communities.
I hope this will help to keep homeservers clean.

### CODE REVIEW

**Comment by krombel** [comment]
@Matombo Please have a look at https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst

**Comment by Matombo** [comment]
Done. Thanks for pointing me to the contribution readme.
First time i rebase something, hope i didn't break anything (one commit message now shows up twice?).

Developer Certificate of Origin
Version 1.1
Signed-off-by: Werner Sembach <werner.sembach at fau.de>

File: scripts-dev/nuke-community-from-db.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## This script will remove (hopefully) all trace of the given Community ID from
+## your homeserver.db
+
+## Do not run it lightly.
+
+set -e
+
+if [ "$1" == "-h" ] || [ "$1" == "" ]; then
+  echo "Call with group_id (Community ID e.g. +test:example.com) as first option and then pipe it into the database. So for instance you might run"
+  echo " nuke-room-from-db.sh <group_id> | sqlite3 homeserver.db"
```
Comment (richvdh) [inline scripts-dev/nuke-community-from-db.sh]: `nuke-community-from-db.sh`

**Comment by richvdh** [comment]
@erikjohnston you're more familiar with the groups stuff than me - is this a plausible thing to do?

**Comment by erikjohnston** [comment]
This looks sane for local groups with local users, but any remote users won't get told the groups has been deleted from under them, so not 100% sure what their experience is

**Comment by richvdh** [comment]
(I think we'd want anything like this to be in contrib rather than scripts-dev)

**Comment by Matombo** [comment]
(Done)

**Comment by aaronraimist** [comment]
There is now an admin API to delete groups, does that fix this? https://github.com/matrix-org/synapse/blob/master/docs/admin_api/delete_group.md

**Comment by Matombo** [comment]
Yep, sounds like it's doing exactly what this script is doing.

**Pull request closed without merging**